R/checkers.R

```diff
@@ -65,9 +65,11 @@ check_sums_to_one <- function(x,
 #' @name type_checker
 #'
 #' @return NULL, invisibly
+#' @noRd
 NULL
 
 #' @rdname type_checker
+#' @noRd
```

Comment:
my personal preference is to doc them and use @keywords internal but I don't feel strongly

Reply:
I like this feedback in general but not for these functions in particular. I'm keeping this PR as draft until we merge the big stacked one and then I'll do that for some of those functions.

Reply:
sounds reasonable. issue?